packages/SystemsThinkingWorkbench-Core.package/STWConnection.class/class/vertices.color.borderWidth.borderColor..st

```diff
@@ -0,0 +1,3 @@
+instance creation
+vertices: verts color: c borderWidth: bw borderColor: bc
```

Comment:
1. Physics code, please use more sensible names.
2. Do we need this method, or could beStraightSegments not be called in `from:to:color:width:`?